README.rst

```diff
@@ -29,24 +29,23 @@ Corresponding Python packages are available via PyPi:
 Documentation
 ~~~~~~~~~~~~~
 
-Read the fine manual (RTFM!) at http://easybuild.readthedocs.org .
+Read the fine manual (RTFM!) at http://docs.easybuild.io .
 
 Getting started
 ~~~~~~~~~~~~~~~
 
-The recommended way of installing EasyBuild is via the
-`documented bootstrap procedure <http://easybuild.readthedocs.org/en/latest/Installation.html#bootstrapping-procedure>`_
-You should `configure <http://easybuild.readthedocs.org/en/latest/Configuration.html>`_
+The recommended way of installing EasyBuild is `using pip <https://docs.easybuild.io/en/latest/Installation.html>`_.
+You should `configure <http://docs.easybuild.io/en/latest/Configuration.html>`_
```

Comment:
After we flip the switch to the new MkDocs-based documentation, we should replace these URLs again, although the `.../en/latest/...` will work fine (there are auto-redirects in place)